src/jit/compiler.cpp

```diff
@@ -8297,10 +8297,10 @@ void dumpConvertedVarSet(Compiler* comp, VARSET_VALARG_TP vars)
     pVarNumSet            = (BYTE*)_alloca(varNumSetBytes);
     memset(pVarNumSet, 0, varNumSetBytes); // empty the set
 
-    VARSET_ITER_INIT(comp, iter, vars, varIndex);
-    while (iter.NextElem(&varIndex))
+    VarSetOps::Iter iter(comp, vars);
+    for (int i = iter.NextElem(); i != -1; i = iter.NextElem())
     {
```

Comment:
IMO, you should leave the descriptive `varIndex` name and not replace it with `i`. Same for other cases where you've done this.